app/models/lti_configuration.rb

```diff
@@ -3,6 +3,14 @@
 class LtiConfiguration < ApplicationRecord
   belongs_to :organization
 
+  enum lms_type: [
+    :other,
```

Comment:
Nit: Is this the order which we would display them in? Usually we would put an `unknown` or `other` option at the end of an enum

Reply:
Yea however I also display a default. 

https://github.com/education/classroom/blob/b63bb2d26aafe1d372cf8c04ffcac5c103b23188/db/schema.rb#L176

To make it easier to know where the default is, I just made it 0. 

I can still fix this to add it to the end, but because this is going to be changing (adding more LMSs) I think it sits better at the top.

Reply:
We should probably declare the enum as a hash with explicit orderings rather than array, as that way we're not dependent on implicit ordering: 

https://naturaily.com/blog/ruby-on-rails-enum (see section 1)